lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -519,19 +519,45 @@ function dosomething_campaign_get_campaigns_doing($uid = NULL) {
   }
 
   // Get all signups.
-  $signup_nids = dosomething_signup_get_signup_nids_by_uid($uid);
+  if (variable_get('rogue_collection', FALSE)) {
+    $northstar_id = dosomething_user_get_northstar_id($uid);
+
+    $rogue_signups = dosomething_rogue_get_activity([
+      'filter' => [
+        'northstar_id' => $northstar_id,
+      ],
+      'limit' => 300,
```

Comment:
@katiecrane ohhh you're right ! I don't need to add this limit here - it works automatically to format and the view more button works too!